themes/everforest_dark.yml

```diff
@@ -0,0 +1,28 @@
+# Colors (Everforest Dark)
+colors:
+  # Default colors
+  primary: 
+    background: '0x2d353b'
+    foreground: '0xd3c6aa'
+
+  # Normal colors
+  normal:
+    black:   '0x475258'
+    red:     '0xe67e80'
+    green:   '0xa7c080'
+    yellow:  '0xdbbc7f'
+    blue:    '0x7fbbb3'
+    magenta: '0xd699b6'
+    cyan:    '0x83c092'
+    white:   '0xd3c6aa'
+  
+  # Bright colors
+  bright:
+    black:   '0x475258'
+    red:     '0xe67e80'
+    green:   '0xa7c080'
+    yellow:  '0xdbbc7f'
+    blue:    '0x7fbbb3'
+    magenta: '0xd699b6'
+    cyan:    '0x83c092'
+    white:   '0xd3c6aa'
```

Comment:
Could you add a newline to the end of this file just for consistency? Don't want people to randomly modify this file just by opening it because their editor automatically appends newlines.

Reply:
Of course. Done. Thank you!